build_projects/Microsoft.DotNet.Cli.Build.Framework/TargetConditions/BuildPlatformsAttribute.cs

```diff
@@ -18,11 +19,17 @@ public BuildPlatformsAttribute(params BuildPlatform[] platforms)
             _buildPlatforms = platforms;
         }
 
+        public BuildPlatformsAttribute(BuildPlatform platform, string version)
+        {
+            _buildPlatforms = new BuildPlatform[] { platform };
+            _version = version;
+        }
+
         public override bool EvaluateCondition()
         {
             foreach (var platform in _buildPlatforms)
             {
-                if (CurrentPlatform.IsPlatform(platform))
+                if (CurrentPlatform.IsPlatform(platform, _version))
```

Comment:
This doesn't seem right.  If you had multiple BuildPlatforms, there is no way they have the same version.

Maybe BuildPlatformsAttribute needs to "AllowMultilpe=true", and then change it so a single BuildPlatforms attribute takes in a single platform and version value.